staging/src/k8s.io/sample-apiserver/docs/minikube-walkthrough.md

```diff
@@ -1,56 +1,67 @@
 # Minikube walkthrough
 
 This document will take you through setting up and trying the sample apiserver on a local minikube from a fresh clone of this repo.
+You can optionally run and develop the sample apiserver on your local machine and use minikube to host the etcd.
 
-## Pre requisites
+## Prerequisites
 
-- Go 1.7.x or later installed and setup. More infomration can be found at [go installation](https://golang.org/doc/install)
+- Go 1.7.x or later installed and setup. More information can be found at [go installation](https://golang.org/doc/install)
```

Comment:
Honestly, I would drop the whole sentence. Having Go installed for a Go project is pretty obvious. Moreover, Go 1.7.x. is probably pretty outdated. Not even sure it still works with our libraries.